openbb_terminal/stocks/comparison_analysis/ca_controller.py

```diff
@@ -737,7 +737,8 @@ def call_cashflow(self, other_args: List[str]):
             dest="s_timeframe",
             type=str,
             default=None,
-            help="Specify yearly/quarterly timeframe. Default is last.",
+            help="Specify year/quarter of the cashflow statement to be retrieved. The format for year is YYYY and for "
```

Comment:
Technically its not quarter, since specifying year quarter would be something like 2021/Q3, but this requires the quarter end date.  Not sure if there is an easy workaround other than having a list of all possible dates ?

Reply:
The problem is that different firms publish their quarterly report on different dates, so you cannot really have autocomplete with this.
One option is to change the quarter flag to having int representing the quarter to retrieve, then we only need to enter the year in the timeframe flag. However this requires extensive refactoring and other commands use the same functions too, thus they would also have to be changed.

Reply:
Yeah this will require some thinking and we should probably also move away from using this scraping source moving forward.  But I am good with how it is now.